### ISSUE TITLE
Single line labels

### ISSUE THREAD

**Issue by strarsis** [CalderaWP/Caldera-Forms#2067]
It would be great if form labels can be next to their input fields (single line) - 
instead being on a separate line, above the input field.
Many form layouts have the label next to input.

**Comment by Shelob9** [comment]
Yes, I agree field label position should be easy to change. Though I think
one option for field is bad UX. Some other form plugins have label position
as a setting per field, I don't like that, I'd rather have a global setting
for the form, maybe a hook per field... I could be wrong and am open to
other opinions.

**Comment by strarsis** [comment]
@Shelob9: In the end IMHO this should be just representation/styling - a different stylesheet or applied class on the form element (or maybe also on particular fields for finer granularity) that is applied which causes the layout (e.g. flexbox). The markup should make this as easy as possible though (including the "classical" problems of aligning labels of different widths).

**Comment by Shelob9** [comment]
I have not thaught this out as far as implimention. Your approach sounds
good. I must admit to not knowing much about flexbox.

If you have time, I'd love to see a mockup in a code pen or something :)

**Comment by strarsis** [comment]
Starting with a basic pen: https://codepen.io/strarsis/pen/gXmmMY
